cylc/uiserver/schema.py

```diff
@@ -150,9 +150,7 @@ class Arguments:
                 this cycle point.
             ''')
         )
-        mode = RunMode(
-            default_value=RunMode.Live.value  # type: ignore
-        )
+        mode = RunMode()
```

Comment:
This default value was causing client-server errors, not sure why, removing for the beta.